test/collect_coverage_api_test.dart

```diff
@@ -92,6 +101,11 @@ Future<Map<String, dynamic>> _collectCoverage(
     }
   });
   final Uri serviceUri = await serviceUriCompleter.future;
+  Set isolateIdSet;
```

Comment:
Add the generic to the type annotation here.

Reply:
Done 👍